src/main/resources/com/tresys/nitf/xsd/nitf.dfdl.xsd

```diff
@@ -778,7 +778,10 @@ The message root is 'NITF'.
 
   <xs:group name="PadOutputPixelCodeLeadingSpace">
     <xs:sequence>
-      <xs:element name="LeadingSpace" type="xs:int" dfdl:lengthKind="explicit" dfdl:length="{ if (../../../../Header/PixelJustification eq 'L') then 0 else (xs:int(fn:ceiling(../../PadOutputPixelCodeLength div 8.0) * 8) - ../../PadOutputPixelCodeLength) }" dfdl:lengthUnits="bits" dfdl:representation="binary" dfdl:outputValueCalc="{ 0 }"/>
+      <xs:element name="LeadingSpace" type="xs:string" dfdl:encoding="X-DFDL-BITS-MSBF" dfdl:lengthKind="explicit"
+          dfdl:length="{ if (../../../../Header/PixelJustification eq 'L') then 0 else (xs:int(fn:ceiling(../../PadOutputPixelCodeLength div 8.0) * 8) - ../../PadOutputPixelCodeLength) }"
+          dfdl:lengthUnits="characters" dfdl:representation="text" dfdl:truncateSpecifiedLengthString="yes"
+          dfdl:outputValueCalc="{ '00' }"/>
```

Comment:
I think your outputValueCalc should be '00000000' (8 zero bits) which is I believe the max length of this LeadingSpace quasi-field. 

As now, if the length evaluates to 6, I think it will fail because you would need padding (with zeros) to be done to lengthen the string. I don't see the properties for that. You could add them, but I think just using '00000000' as the value eliminates the problem? 

I'm actually not sure it would fail. Maybe if the length turned out to be 6 it would output the 2 bits you have, and the rest would be filled from the fillByte, which is probably '%#r00;' since this is binary data so it would still "work".

Reply:
Personally I like using `00000000` better, feels a bit more consistent so we don't have part of the field being BITS-MSBF encoded string and the rest being fill byte.